api/app/Http/Middleware/AuditQueryMiddleware.php

```diff
@@ -36,6 +36,21 @@ public function handle(Request $request, Closure $next)
             );
         }
 
-        return $next($request);
+        $start = hrtime(true);
+        $result = $next($request);
+        $end = hrtime(true);
+        $elapsedTime = ($end - $start) / 1000000000;
```

Comment:
Would recommend calling this elapsedSeconds to make the units clear.

Reply:
Aye, and moved into its own file in https://github.com/GCTC-NTGC/gc-digital-talent/pull/9045/commits/72ac439970b370ba6b497438ec2296f4ceed3845